src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figures {
```

Comment:
так цей тип перейде автоматично усім наслідникам
```suggestion
public abstract class Figures implements Behaviour {
```